pkg/util/k8sutil/k8sutil.go

```diff
@@ -192,10 +193,16 @@ func NewFromConfig(cfg *rest.Config) (KubernetesClient, error) {
 }
 
 // SetPostgresCRDStatus of Postgres cluster
-func (client *KubernetesClient) SetPostgresCRDStatus(clusterName spec.NamespacedName, status string) (*apiacidv1.Postgresql, error) {
+func (client *KubernetesClient) SetPostgresCRDStatus(clusterName spec.NamespacedName, status string, numberOfInstances int32, labelSelector string, observedGeneration int64, existingConditions apiacidv1.Conditions, message string) (*apiacidv1.Postgresql, error) {
```

Comment:
Instead of extending the list of arguments we could also pass a PostgresStatus struct, right?

Reply:
Yeah, made modifications to use the PostgresStatus struct